auth/src/main/java/org/aerogear/mobile/auth/AuthService.java

```diff
@@ -102,9 +102,9 @@ public void configure(final MobileCore core, final ServiceConfiguration serviceC
      * @param context
      */
     public void init(final Context context, final AuthServiceConfiguration authServiceConfiguration) {
```

Comment:
so both `configure` and `init` methods need to be called to initialise the `AuthService` instance properly. I think we should add checks inside the public methods (`currentUser`, `login`, `logout`) to verify that the instance has been initialised properly. Otherwise it should throw an exception and explain that the instance hasn't been inited properly.

Reply:
Addressed